lightning/src/ln/channel.rs

```diff
@@ -1093,6 +1094,14 @@ impl<Signer: Sign> Channel<Signer> {
 		// of 1.1 sat/vbyte and a receiver that wants 1.1 rounded up to 2. Thus, we always add 250
 		// sat/kw before the comparison here.
 		if feerate_per_kw + 250 < lower_limit {
+			if let Some(cur_feerate) = cur_feerate_per_kw {
+				if feerate_per_kw > cur_feerate {
+					log_error!(logger,
```

Comment:
Is there a reason we're doing this at the `error` level currently and not just `warn`? It seems to imply something will fail, but we're not failing anything at this stage. Is it just for the future's sake when we do start failing this channel and don't need to promote it to an error then?